Gemfile.lock

```diff
@@ -685,7 +685,7 @@ DEPENDENCIES
   pry-rails
   pry-rescue
   pry-stack_explorer
-  puma (= 4.3.6)
+  puma
```

Comment:
This change is not related to the PR. :)

Reply:
yeah, someone changed gemfile and did not bundle